packages/web/src/components/CheckboxField/_theme.scss

```diff
@@ -0,0 +1,24 @@
+$input-checked-disabled-background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTkiIHZpZXdCb3g9IjAgMCAxOCAxOSIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTE2IDAuMzQ0NzI3SDJDMC45IDAuMzQ0NzI3IDAgMS4yNDQ3MyAwIDIuMzQ0NzNWMTYuMzQ0N0MwIDE3LjQ0NDcgMC45IDE4LjM0NDcgMiAxOC4zNDQ3SDE2QzE3LjEgMTguMzQ0NyAxOCAxNy40NDQ3IDE4IDE2LjM0NDdWMi4zNDQ3M0MxOCAxLjI0NDczIDE3LjEgMC4zNDQ3MjcgMTYgMC4zNDQ3MjdaTTcuNzEgMTMuNjM0N0M3LjMyIDE0LjAyNDcgNi42OSAxNC4wMjQ3IDYuMyAxMy42MzQ3TDIuNzEgMTAuMDQ0N0MyLjMyIDkuNjU0NzMgMi4zMiA5LjAyNDczIDIuNzEgOC42MzQ3M0MzLjEgOC4yNDQ3MyAzLjczIDguMjQ0NzMgNC4xMiA4LjYzNDczTDcgMTEuNTE0N0wxMy44OCA0LjYzNDczQzE0LjI3IDQuMjQ0NzMgMTQuOSA0LjI0NDczIDE1LjI5IDQuNjM0NzNDMTUuNjggNS4wMjQ3MyAxNS42OCA1LjY1NDczIDE1LjI5IDYuMDQ0NzNMNy43MSAxMy42MzQ3WiIgZmlsbD0iI0M0QzRDNCIvPgo8L3N2Zz4K');
+$input-unchecked-background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTgiIHZpZXdCb3g9IjAgMCAxOCAxOCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTE2IDJWMTZIMlYySDE2Wk0xNiAwSDJDMC45IDAgMCAwLjkgMCAyVjE2QzAgMTcuMSAwLjkgMTggMiAxOEgxNkMxNy4xIDE4IDE4IDE3LjEgMTggMTZWMkMxOCAwLjkgMTcuMSAwIDE2IDBaIiBmaWxsPSIjMTMyOTMwIi8+Cjwvc3ZnPgo=');
+$input-unchecked-disabled-background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTgiIHZpZXdCb3g9IjAgMCAxOCAxOCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTE2IDJWMTZIMlYySDE2Wk0xNiAwSDJDMC45IDAgMCAwLjkgMCAyVjE2QzAgMTcuMSAwLjkgMTggMiAxOEgxNkMxNy4xIDE4IDE4IDE3LjEgMTggMTZWMkMxOCAwLjkgMTcuMSAwIDE2IDBaIiBmaWxsPSIjQzRDNEM0Ii8+Cjwvc3ZnPgo=');
+$input-indeterminate-background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTgiIHZpZXdCb3g9IjAgMCAxOCAxOCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTE2IDBIMkMwLjkgMCAwIDAuOSAwIDJWMTZDMCAxNy4xIDAuOSAxOCAyIDE4SDE2QzE3LjEgMTggMTggMTcuMSAxOCAxNlYyQzE4IDAuOSAxNy4xIDAgMTYgMFpNMTQgMTBINFY4SDE0VjEwWiIgZmlsbD0iIzI5NjE2RiIvPgo8L3N2Zz4K');
+$input-indeterminate-disabled-background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTgiIHZpZXdCb3g9IjAgMCAxOCAxOCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTE2IDBIMkMwLjkgMCAwIDAuOSAwIDJWMTZDMCAxNy4xIDAuOSAxOCAyIDE4SDE2QzE3LjEgMTggMTggMTcuMSAxOCAxNlYyQzE4IDAuOSAxNy4xIDAgMTYgMFpNMTQgMTBINFY4SDE0VjEwWiIgZmlsbD0iI0M0QzRDNCIvPgo8L3N2Zz4K');
```

Comment:
All these backgrounds can be changed into CSS. All that really needs to be (or is better to be) an SVG is the tick itself, maybe the indeterminate symbol. That way it will be easier to style it and our CSS will also be smaller.

Reply:
BTW, having inline SVG here, we would benefit from its readability and even smaller size.

See https://github.com/react-ui-org/react-ui/blob/master/src/lib/styles/theme-constants/_svg.scss

We can surely do this later as it requires additional [Sass tooling](https://github.com/react-ui-org/react-ui/blob/master/src/lib/styles/tools/_svg.scss).

Reply:
Ok, fixed.